packages/v-protocol/README.md

```diff
@@ -4,65 +4,9 @@
-
-<!-- ## Installation
-    To install vertica-nodejs with npm: ``` TO DO ```
-
-    To use vertica-nodejs linked locally from source (not recommended in production): ``` TO DO - Take notes from http://confluence.verticacorp.com/display/DEV/Node.js+Development+Resources```
-
--->
-
-### Features
-
-- Pure JavaScript client and native libpq bindings share _the same API_
-- Connection pooling
-- Extensible JS ↔ Vertica data-type coercion
-<!-- - Supported Vertica features -->
-  <!-- - Async notifications with `LISTEN/NOTIFY` verifiy this -->
-  <!-- - Bulk import & export with `COPY TO/COPY FROM` not part of the MVP -->
-
-## Support
-
-vertica-nodejs is free software. If you encounter a bug with the library please open an issue on the [GitHub repo](https://github.com/vertica/vertica-nodejs). If you have questions unanswered by the documentation please open an issue pointing out how the documentation was unclear and we will address it as needed. 
-
-When you open an issue please provide:
-
-- version of Node
-- version of Vertica
-- smallest possible snippet of code to reproduce the problem
-
-<!-- 
-## Contributing
-
-Outside contributions to this project are greatly appreciated. Following standard Vertica open source practices, please see [CONTRIBUTING.md](CONTRIBUTING.md)
--->
-
-## Examples
-
-See the [examples directory](https://github.com/vertica/vertica-nodejs/tree/master/examples) for sample applications showing how to use the vertica-nodejs client driver.
-
-## Setting up for local development
-
-1. Clone the repo
-2. From your workspace root run `yarn` and then `yarn lerna bootstrap`
-3. Ensure you have a Vertica instance running with 
-4. Ensure you have the proper environment variables configured for connecting to the instance (`V_HOST`, `V_PORT`, `V_USER`, `V_PASSWORD`, `V_DATABASE`, `V_BACKUP_SERVER_NODE`)
-5. Run `yarn test` to run all the tests, or run `yarn test` from within an individual package to only run that package's tests
-
-If using VS Code, you can install the `Remote - Containers` extension and it will use the configuration under the `.devcontainer` folder to automatically create dev containers, including Vertica.  See [here](https://code.visualstudio.com/docs/remote/containers) for more information on developing in containers using VS Code.  This process will complete steps 2 to 4 above.
-
-## Troubleshooting and FAQ
-
-The causes and solutions to common errors can be found among the [Frequently Asked Questions (FAQ)](https://github.com/vertica/vertica-nodejs/wiki/FAQ)
+Vertica protocol code used to send and receive messages defined in the Vertica protocol.
+This is used by the [vertica-nodejs](https://github.com/vertica/vertica-nodejs) client driver and is *not* meant to be
+used by developers writing their own applications with vertica-nodejs.
 
 ## License
```

Comment:
The license under the top-level README has been changed to Apache 2.0, should we do the same for this (and the other packages)?

Reply:
+1

Reply:
Looks like we still have the MIT license in each of the readme.md files in each package (v-connection-string, v-pool, v-protocol and vertica-nodejs).   Will you be changing that Danny?

Reply:
I will indeed. vertica-nodejs has already been changed in the open PR